prime-dotnet-webapi/Configuration/Database/Seeded Tables/LicenseDetailConfiguration.cs

```diff
@@ -102,6 +103,9 @@ public override IEnumerable<LicenseDetail> SeedData
                     new LicenseDetail { Id = 87, LicenseCode = 62, Prefix = "98", Manual = false, Validate = true,  NamedInImReg = true,  LicensedToProvideCare = true,  PrescriberIdType = PrescriberIdType.Mandatory, EffectiveDate = ImRegUpdate1Date, CreatedTimeStamp = SEEDING_DATE, UpdatedTimeStamp = SEEDING_DATE },
                     new LicenseDetail { Id = 88, LicenseCode = 63, Prefix = "98", Manual = true,  Validate = true,  NamedInImReg = true,  LicensedToProvideCare = false,                                                EffectiveDate = ImRegUpdate1Date, CreatedTimeStamp = SEEDING_DATE, UpdatedTimeStamp = SEEDING_DATE },
 
+                    new LicenseDetail { Id = 89, LicenseCode = 52, Prefix = "L9", Manual = false, Validate = true,  NamedInImReg = true,  LicensedToProvideCare = true, EffectiveDate = new(2022, 2, 2, 8, 0, 0, DateTimeKind.Utc), CreatedTimeStamp = SEEDING_DATE, UpdatedTimeStamp = SEEDING_DATE },
+                    new LicenseDetail { Id = 90, LicenseCode = 55, Prefix = "L9", Manual = false, Validate = true,  NamedInImReg = true,  LicensedToProvideCare = true, EffectiveDate = new(2022, 2, 2, 8, 0, 0, DateTimeKind.Utc), CreatedTimeStamp = SEEDING_DATE, UpdatedTimeStamp = SEEDING_DATE },
```

Comment:
Why add new license details when the existing ones were in error? Seems more accurate to update the existing ones

Reply:
pushed